gradle/libs.versions.toml

```
@@ -1,5 +1,5 @@
 [versions]
-agp = "7.4.2"
+agp = "7.4.1" # maximum version supported by IDEA 2023.1
```

nit: if we forget to update comment, then this will remain valid on later agp bumps

```suggestion
agp = "7.4.1" # 7.4.1 is maximum version supported by IDEA 2023.1
```